test/test_sparse_csr.py

```diff
@@ -1462,6 +1462,60 @@ def run_test_block_addmm_addmv(self,
         self.assertEqual(actual, out)
         self.assertEqual(actual, expected)
 
+    @parametrize("block_size", [16, 32, 64])
+    @parametrize("index_dtype", [torch.int32, torch.int64])
+    @skipCUDAIfRocm
+    @onlyCUDA
+    @dtypes(torch.half, torch.bfloat16)
+    @dtypesIfCUDA(*[torch.half] if SM53OrLater else [],
+                  *[torch.bfloat16] if SM80OrLater else [])
+    def test_triton_bsr_dense_bmm(self, device, dtype, index_dtype, block_size):
```

Comment:
I don't think we'll have Triton available in our internal test infra, can you guard against that with `@unittest.skipIf(IS_FBCODE and IS_REMOTE_GPU, "Test requires Triton")`?

Reply:
You can import `IS_FBCODE ` and `IS_REMOTE_GPU ` from `torch.testing._internal.common_utils` if it's not already present in this file.